cshsms/settings_travis.py

```diff
@@ -14,3 +16,10 @@
 REMOTE = {'user': 'travis',
           'keyfile': '~/.ssh/travis.pem',
           'host': 'localhost'}
+
+TEXTLOCAL_API = os.getenv("TEXTLOCAL_API")
+TEXTLOCAL_PRIMARY_ID = os.getenv("TEXTLOCAL_PRIMARY_ID")
+TEXTLOCAL_PHONENUMBER = os.getenv("TEXTLOCAL_PHONENUMBER")
+HSPSMS_API = os.getenv("HSPSMS_API")
+HSPSMS_USERNAME = os.getenv("HSPSMS_USERNAME")
+HSPSMS_SENDERNAME = os.getenv("HSPSMS_SENDERNAME")
```

Comment:
If we're no longer running live tests on Travis, we don't need to support these anymore, right?